### PR TITLE
Register Subdomain: Sylestic

### PR DESCRIPTION
**Pull request by ghost**
Added the domain: Sylestic

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry CNAME doesn't work with other records.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Removing the MX records is fine, but your website isn't reachable. Please provide the URL for your website, since `https://sylestic.github.io` is not reachable.

File: domains/Sylestic.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+   "owner":{
+      "username":"Sylestic",
+      "email":"sylestic@protonmail.com"
+   },
+   "record":{
+      "CNAME":"sylestic.github.io"
```
Comment (TheBotlyNoob) [inline domains/Sylestic.json:7]: Maybe you want to redirect it to https://sylestic.ml/?
```suggestion
      "URL":"sylestic.ml"
```

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

File: domains/Sylestic.json (hunk 1 above)
Comment (Sylestic) [reply domains/Sylestic.json]: Thanks!

**Review by wdhdev — APPROVED** [review]

**Comment by Sylestic** [comment]
Hey! Can this be merged?

**Pull request closed without merging**